test/e2e/util.go

```diff
@@ -91,69 +93,67 @@ func initTestContext() *context {
 
 	cxt.enableNamespaceAsQueue = enableNamespaceAsQueue
 
-	_, err = cxt.kubeclient.CoreV1().Namespaces().Create(&v1.Namespace{
+	_, err = cxt.kubeclient.CoreV1().Namespaces().Create(gcontext.Background(), &v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: cxt.namespace,
 		},
-	})
-	Expect(err).NotTo(HaveOccurred())
-
-	_, err = cxt.kubeclient.SchedulingV1beta1().PriorityClasses().Create(&schedv1.PriorityClass{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: masterPriority,
-		},
-		Value:         100,
-		GlobalDefault: false,
-	})
-	Expect(err).NotTo(HaveOccurred())
-
-	_, err = cxt.kubeclient.SchedulingV1beta1().PriorityClasses().Create(&schedv1.PriorityClass{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: workerPriority,
-		},
-		Value:         1,
-		GlobalDefault: false,
-	})
-	Expect(err).NotTo(HaveOccurred())
+	}, metav1.CreateOptions{})
+	//Expect(err).NotTo(HaveOccurred())
+
+	/* 	_, err = cxt.kubeclient.SchedulingV1beta1().PriorityClasses().Create(gcontext.Background(), &schedv1.PriorityClass{
+	   		ObjectMeta: metav1.ObjectMeta{
+	   			Name: masterPriority,
+	   		},
+	   		Value:         100,
+	   		GlobalDefault: false,
+	   	}, metav1.CreateOptions{})
+	   	Expect(err).NotTo(HaveOccurred())
+
+	   	_, err = cxt.kubeclient.SchedulingV1beta1().PriorityClasses().Create(gcontext.Background(), &schedv1.PriorityClass{
+	   		ObjectMeta: metav1.ObjectMeta{
+	   			Name: workerPriority,
+	   		},
+	   		Value:         1,
+	   		GlobalDefault: false,
+	   	}, metav1.CreateOptions{})
+	   	Expect(err).NotTo(HaveOccurred()) */
 
 	return cxt
 }
 
 func namespaceNotExist(ctx *context) wait.ConditionFunc {
 	return func() (bool, error) {
-		_, err := ctx.kubeclient.CoreV1().Namespaces().Get(ctx.namespace, metav1.GetOptions{})
+		_, err := ctx.kubeclient.CoreV1().Namespaces().Get(gcontext.Background(), ctx.namespace, metav1.GetOptions{})
 		if !(err != nil && errors.IsNotFound(err)) {
 			return false, err
 		}
 		return true, nil
 	}
 }
 
-
 func cleanupTestContextExtendedTime(cxt *context, seconds time.Duration) {
-	foreground := metav1.DeletePropagationForeground
-
-	err := cxt.kubeclient.CoreV1().Namespaces().Delete(cxt.namespace, &metav1.DeleteOptions{
+	//foreground := metav1.DeletePropagationForeground
```

Comment:
Recommend to open a work item to added a pointer to an array of AWs to cleanup function.  That way the AWs to be cleaned up can be added after the `defer` statement in the calls.

Reply:
Addressed!!

Reply:
Thank you!